packages/dds/tree/src/core/rebase/types.ts

```diff
@@ -41,6 +41,12 @@ export function mintRevisionTag(): RevisionTag {
 	return generateStableId();
 }
 
+/** Special types of commits */
+export enum GraphCommitType {
+	/** A commit that was generated by an undo. */
+	Undo = "undo",
```

Comment:
I think this will end up in summaries, so we probably should use an integer for performance's sake.